src/eva/tests/config/testCubedSphereRestart.yaml

```diff
@@ -0,0 +1,66 @@
+diagnostics:
+
+  # Data read
+  # ---------
+- data:
+    type: CubedSphereRestart
+    datasets:
+      - name: experiment
+        variable: T
+        resolution: C768
+        fv3_filenames:
+          - ${data_input_path}/20210801.000000.sfc_data.tile1.nc
```

Comment:
hmm, we need to figure out how to download this data for the test, and also go lower resolution... the operational resolution is a bit much for this purpose...